src/faq

```diff
@@ -244,36 +215,36 @@
 
   <h4 id="living-standard">What does “Living Standard” mean?<a class="self-link" href="#living-standard"></a></h4>
 
-  <p>The WHATWG specifications are described as Living Standards. This means that they are standards
-  that are continuously updated as they receive feedback, either from Web designers, browser
+  <p>The WHATWG standards are described as Living Standards. This means that they are standards
+  that are continuously updated as they receive feedback, either from web designers, browser
   vendors, tool vendors, or indeed any other interested party. It also means that new features get
-  added to them over time, at a rate intended to keep the specifications a little ahead of the
-  implementations but not so far ahead that the implementations give up.
+  added to them over time, at a rate intended to keep the standard a little ahead of the
+  implementations, but not so far ahead that the implementations give up.
 
   <p>Despite the continuous maintenance, or maybe we should say <i>as part</i> of the continuing
```

Comment:
I don't want to spend any time debating this, so please disregard my feedback here for merging purposes. But I am idly curious what part of the definition of i you think this matches.